docs/Examples/generate-blocks-and-txs.js

```diff
@@ -0,0 +1,172 @@
+
+    try {
+      const wwit = Boolean(Math.random() < 0.5);
+      const newWallet = await wdb.create({
+        id: wName,
+        witness: wwit
+      });
+
+      wallets.push(newWallet);
+
+      for (const aName of accountNames) {
+        const awit = Boolean(Math.random() < 0.5);
+        await newWallet.createAccount({
+          name: aName,
+          witness: awit
+        });
+      }
+    } catch (e) {
+      console.log(`Error creating wallet ${wName}:`, e.message);
+    }
+  }
+
+  if (!wallets.length) {
+    console.log('No wallets created, likely this script has already been run');
+    return;
+  }
+  accountNames.push('default');
+
+  console.log('Mining initial blocks...');
+  const primary = await wdb.get('primary');
+  const minerReceive = await primary.receiveAddress(0);
+  for (let i = 0; i < numInitBlocks; i++) {
+    await mineRegtestBlockToPast(minerReceive);
+  }
+
+  console.log('Air-dropping funds to the people...');
+  const balance = await primary.getBalance(0);
+
+  const totalAmt = balance.confirmed;
+  const amtPerAcct = Math.floor(
+    totalAmt / (walletNames.length * accountNames.length)
+  );
+
+  const outputs = [];
+  for (const wallet of wallets) {
+    for (const aName of accountNames) {
+      const recAddr = await wallet.receiveAddress(aName);
+      outputs.push({
+        value: amtPerAcct,
+        address: recAddr
+      });
+    }
+  }
+
+  const t = await primary.send({
+    outputs: outputs,
+    rate: feeRate,
+    subtractFee: true
+  });
+
+  console.log('Confirming airdrop...');
+  await mineRegtestBlockToPast(minerReceive);
+
+  console.log('Creating a big mess!...');
+  for (let b = 0; b < numTxBlocks; b++) {
+    for (let t = 0; t < numTxPerBlock; t++) {
+      // Randomly select recipients for this tx
+      const outputs = [];
+      const numOutputs = Math.floor(Math.random() * maxOutputsPerTx) + 1;
+      for (let o = 0; o < numOutputs; o++) {
+        const recWallet = wallets[Math.floor(Math.random() * wallets.length)];
+        const recAcct =
+          accountNames[Math.floor(Math.random() * accountNames.length)];
+        const recAddr = await recWallet.receiveAddress(recAcct);
+        const value = Math.floor(
+          Math.random() * (maxSend - minSend) + minSend / numOutputs
+        );
+
+        outputs.push({
+          value: value,
+          address: recAddr
+        });
+      }
+
+      // Randomly choose a sender for this tx
+      const sendWallet = wallets[Math.floor(Math.random() * wallets.length)];
+      const sendAcct = accountNames[Math.floor(Math.random() * wallets.length)];
+      try {
+        await sendWallet.send({
+          account: sendAcct,
+          outputs: outputs,
+          rate: feeRate,
+          subtractFee: true
+        });
+      } catch (e) {
+        console.log(`Problem sending tx: ${e}`);
+      }
+    }
+
+    // CONFIRM
+    await mineRegtestBlockToPast(minerReceive);
+  }
+
+  console.log('All done! Go play.');
+})();
```

Comment:
There could be a `.catch()` here, as similar to https://github.com/bcoin-org/bcoin/blob/master/bin/node#L56-L59 and will likely not need the previous try/catch statements.

Reply:
Ah right but as you discovered, I don't expect every transaction to work, so I put the `send`'s in their own try/catch. And the try/catch around wallet creation... that made more sense when I ran this script off disk. If you've already run it once, the wallets will all be created already and you'll get an error trying to make them again. So the idea in both cases is to continue executing.

Reply:
Hmmm...

Reply:
It's likely a timing issue.